pkg/common/host_utils.go

```diff
@@ -34,11 +34,29 @@ import (
 
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"k8s.io/kubernetes/pkg/util/mount"
+	"k8s.io/mount-utils"
 )
 
 const LOOP_CTL_GET_FREE = 0x4C82
 
+var (
+	defaultMountCheckTimeout time.Duration = 50 * time.Second // Default timeout for checking mount status
+)
+
+func init() {
+	// Read environment variables for mount check timeout
+	mountCheckTimeoutStr := os.Getenv("MOUNT_CHECK_TIMEOUT")
```

Comment:
This variable can be set in config map, that will be used to terminate the check for a mount point.